pyLSV2/client.py

```diff
@@ -1967,15 +1968,17 @@ def split_dataset(data):
 
                 temp = "".join('{:02x}'.format(x) for x in data_sub_set[10:])
 
-                print("R_OP section %d: %s %s" % (i, temp, signal_list[i]))
+                # print("R_OP section %d: %s %s" % (i, temp, signal_list[i]))
 
                 # TODO: starts with a string containing the unit of this signal. eg mm or mm/min ...
 
         else:
-            print(
-                "R_OP dataset has unexpected length %d of %s"
-                % (len(data_set), data_set)
-            )
+            #print(
+            #    "R_OP dataset has unexpected length %d of %s"
+            #    % (len(data_set), data_set)
+            #)
+            pass
+
 
```

Comment:
just leaving pass here makes the else clause unnecessary. Instead it should raise an exception like LSV2ProtocolException to interrupt the execution

Reply:
Restored back as it was.